plugin/evm/vm_test.go

```diff
@@ -3926,3 +3931,176 @@ func TestCreateHandlers(t *testing.T) {
 		require.ErrorIs(t, elem.Error, rpc.ErrMissingBatchResponse)
 	}
 }
+
+// newSignedLegacyTx builds a legacy transaction and signs it using the
+// LatestSigner derived from the provided chain config.
+func newSignedLegacyTx(
```

Comment:
this function was introduced in the original delegate call PR, and touched a bunch of other tests. due to how subnet-evm is not synced/up to date with coreth, making those other changes is very difficult / impossible without further work. We include it here for use in the test introduced, but leave the other changes to a future sync.